src/coreclr/hosts/unixcoreconsole/CMakeLists.txt

```diff
@@ -31,8 +31,4 @@ if(NOT CLR_CMAKE_PLATFORM_ANDROID)
   )
 endif()
 
-add_dependencies(coreconsole
-    coreclr
```

Comment:
It is true that these dependencies were not really needed, since they are not build time, but rather runtime dependencies. But that makes me wonder - why do we build corerun and coreconsole during cross components build? I have tried to exclude the add_subdirectory(src/coreclr/hosts) for NOT DEFINED CLR_CROSS_COMPONENTS_BUILD and the build completed successfuly.